File: app/src/main/java/com/erotc/learning/fragment/AboutFragment.java
package com.erotc.learning.fragment;


import android.os.Bundle;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.erotc.learning.BuildConfig;
import com.erotc.learning.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class AboutFragment extends Fragment {


    public AboutFragment() {
        // Required empty public constructor
    }

    public static AboutFragment newInstance() {
        return new AboutFragment();
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_about, container, false);

        TextView about = view.findViewById(R.id.text_version);

        about.setText(String.format(getString(R.string.label_version), BuildConfig.VERSION_NAME));

        return view;
    }
}

File: app/src/main/java/com/erotc/learning/fragment/AboutFragment.kt
package com.erotc.learning.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment
import com.erotc.learning.BuildConfig
import com.erotc.learning.R

/**
 * A simple [Fragment] subclass.
 */
class AboutFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_about, container, false)
        val about = view.findViewById<TextView>(R.id.text_version)
        about.text = String.format(getString(R.string.label_version), BuildConfig.VERSION_NAME)
        return view
    }

    companion object {
        fun newInstance(): AboutFragment {
            return AboutFragment()
        }
    }
}